jira/resources.py

```diff
@@ -482,6 +482,13 @@ def permalink(self):
     def __eq__(self, other):
         return self.id == other.id
 
+    def __getitem__(self, item):
+        # Grab the JIRA server
+        jira_server = self._session.jira_server
+
+        # Look for the appropriate fields list on the server
+        return getattr(self.fields, jira_server._fields[item])
```

Comment:
One other thing to think about is a field named "Release Note". How to handle the use of spaces and other punctuation? Just replace with dashes and/or underscores? Or just remove spaces?